components/Navbar.vue

```diff
@@ -79,6 +79,14 @@
         id="NavChainSelect"
         class="navbar-chain custom-navbar-item"
         data-cy="chain-select" />
+
+      <MobileExpandableSection
+        v-if="isMobile"
```

Comment:
Could you please change the order between `ChainSelectDropdown` and `MobileExpandableSection`? In this way, we could use `v-else` instead.